browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -321,6 +356,11 @@ class MarkdownNoteDetail extends React.Component {
                 </g>
               </svg>
             </button>
+            <button styleName='control-fullScreenButton'
+              onMouseDown={(e) => this.handleFullScreenButton(e)}
+            >
+              <i className={'fa fa-arrows-alt'} styleName='fullScreen-button' />
```

Comment:
haha totally tunnel visioned what you said and didn't realize you mean take off the {}